notebooks/data-sources/Bugzilla/bugzilla_EDA.ipynb

```diff
@@ -0,0 +1,3552 @@
+    "\n",
+    "# sanity check\n",
+    "job_tests_dict._getvalue()[\n",
+    "    (\n",
+    "        '\"redhat-openshift-ocp-release-4.2-informing\"',\n",
+    "        \"periodic-ci-openshift-release-master-ci-4.2-e2e-gcp\",\n",
+    "    )\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c85bc0a0",
+   "metadata": {},
+   "source": [
+    "### Get Linked Bugs under each Test for a Given Dashboard"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "4b6bf2a3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get bugs linked at timestamps up to this amount of time before today\n",
+    "max_age = \"336h\"\n",
+    "\n",
+    "# ci details search url\n",
+    "url = \"https://search.ci.openshift.org/\"\n",
+    "\n",
+    "sample_dashboard = '\"redhat-openshift-ocp-release-4.2-informing\"'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "4b009084",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(<DictProxy object, typeid 'dict' at 0x7fd6594eba90>,\n",
+       " ('\"redhat-openshift-ocp-release-4.2-informing\"',\n",
+       "  'periodic-ci-openshift-release-master-nightly-4.2-e2e-azure',\n",
+       "  'Operator results.operator conditions monitoring'))"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# dict where key is (dashboard, job, test), value is list of related bugs\n",
+    "djt_linked_bugs_dict = manager.dict()\n",
+    "\n",
+    "\n",
+    "def get_bugs_in_test(args_tuple):\n",
+    "    \"\"\"Gets linked and associated bugs for each test+job.\n",
+    "\n",
+    "    Queries the search.ci.openshift website just like the sippy setup does in\n",
+    "    its findBug function defined here:\n",
+    "    https://github.com/openshift/sippy/blob/1a44268082fc600d69771f95a96b4132c9b84285/pkg/buganalysis/cache.go#L230\n",
+    "\n",
+    "    :param jt_dict_dj_pair_tuple: (tuple) Tuple where the first element is the\n",
+    "    shared dict where the result is to be stored and the second element is a\n",
+    "    tuple of (dashboard, job, test)\n",
+    "\n",
+    "    NOTE: If we want to have tqdm with a multiprocessing Pool, we need to use\n",
+    "    pool.imap and thus have only one arg passed. Otherwise we can also split\n",
+    "    the args into separate variables\n",
+    "    \"\"\"\n",
+    "    # unpack\n",
+    "    djt_linked_bugs, djt_tuple = args_tuple\n",
+    "\n",
+    "    # search for linked and associated bugs for this test\n",
+    "    # DO NOT AJAX,MOBILE. THIS HACK PREVENTS REQUEST TIME OUT.\n",
+    "    # read more here - https://stackoverflow.com/a/63377265/9743348\n",
+    "    response = requests.post(\n",
+    "        \"https://search.ci.openshift.org/\",\n",
+    "        data={\n",
+    "            \"type\": \"bug+junit\",\n",
```

Comment:
iirc we _had_ to set it to "bug+junit", otherwise we were unable to tell whether the bugs returned by search.ci.openshift are related to a particular job or not

Reply:
:+1: